### PR TITLE
No conflicts with PRAW's updated version

### PR DESCRIPTION
**Pull request by xCROv**
My commits are just adding to PUNI's supported dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Built to interact with the user notes data store from the reddit moderator
 toolbox ([spec here](https://github.com/creesch/reddit-moderator-toolbox/wiki/JSON:-usernotes)).
 
 **Requirements**:
-* [PRAW](https://github.com/praw-dev/praw)
+* [PRAW](https://github.com/praw-dev/praw)(Supports up to praw-7.1.0)
```
Comment (danthedaniel) [inline README.md:13]:
```suggestion
* [PRAW](https://github.com/praw-dev/praw) (Supports up to praw-7.1.0)
```
Just needs a little spacing

File: puni/version.py [hunk 2]
```diff
@@ -14,4 +14,4 @@
 with puni. If not, see http://www.gnu.org/licenses/.
 """
 
-__version__ = '1.1.2'
+__version__ = '1.1.4'
```
Comment (danthedaniel) [inline puni/version.py:17]:
```suggestion
__version__ = '1.1.3'
```

The version should only increment by 1

**Review by danthedaniel — CHANGES_REQUESTED** [review]
Hey, thanks so much for your pull request! It looks like there are two changes here:

1. Allowing a higher PRAW version
2. Supporting custom warning labels

This code was written a few years ago and I feel confident I could do a much better job now than before. I don't want to rob you of any fun so if you want to edit the PR (or open a new one) with just the PRAW version limit change I'll merge that and release it on PyPI. The other change is something I'd prefer to do in the context of a quick refactor on my own.

**Comment by xCROv** [comment]
> 
> 
> Hey, thanks so much for your pull request! It looks like there are two changes here:
> 
>     1. Allowing a higher PRAW version
> 
>     2. Supporting custom warning labels
> 
> 
> This code was written a few years ago and I feel confident I could do a much better job now than before. I don't want to rob you of any fun so if you want to edit the PR (or open a new one) with just the PRAW version limit change I'll merge that and release it on PyPI. The other change is something I'd prefer to do in the context of a quick refactor on my own.

I think the changes for updating the version are trivial like you said and don't necessarily need a pull request unless you would like one made. My goal was just making sure users installing from Pip or pypi have a working repo.

I fully look forward to following the project if you do refactor it. Thanks a bunch for your effort on this.

**Pull request closed without merging**